src/js_realm_object.hpp

```diff
@@ -110,6 +111,7 @@ struct RealmObjectClass : ClassDefinition<T, realm::js::RealmObject<T>> {
         {"linkingObjectsCount", wrap<linking_objects_count>},
         {"_isSameObject", wrap<is_same_object>},
         {"_objectKey", wrap<get_object_key>},
+        {"_tableKey", wrap<get_table_key>},
```

Comment:
I am exposing `_tableKey` so objects can be stored in cache according to table and object keys; this is what we decided would be good when working on this in bindgen. Let me know if a different approach could be better.

Reply:
It makes a lot of sense as the pair (table key, object key) should be unique within a Realm file.